google-cloud-dlp/src/main/java/com/google/cloud/dlp/v2beta1/ColorUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.dlp.v2beta1;
+
+import com.google.privacy.dlp.v2beta1.Color;
+
+public class ColorUtil {
+
+  private ColorUtil() {}
+
+  public static final Color convert(java.awt.Color c) {
```

Comment:
I think the name is a little too generic. Maybe `toRpcColor`? That way `toAwtColor` can be the reverse of this and we can add more conversions later. @garrettjonesgoogle Do you have an opinion?

Reply:
Yeah, maybe `asDlpColor` might be appropriate.

Reply:
Done